app/src/main/res/layout/fragment_decode.xml

```diff
@@ -67,8 +45,7 @@
 				android:layout_marginRight="16dp"
 				android:layout_marginBottom="16dp"
 				android:typeface="monospace"
-				android:textSize="12sp"
-				tools:text="54 65 73 74 20 51 52 20  Test QR\n43 6F 64 65              Code"/>
```

Comment:
Why remove the tools:text?

Reply:
Good question 🤔 I suppose it was removed by accident. https://github.com/markusfisch/BinaryEye/commit/d0f0bedae2eebe8ce3cfdf1c72d06cfcbf435057 restores it.

Reply:
Actually it was by accident, but I can't see any reason having it because it gets replaced by the code at run-time anyways ?!

Reply:
@molikuner that's `android:text` IIRC; the `tools:` namespace provides mock data for the UI designer in Android Studio

Reply:
Ah thanks for that info. Didn't knew that.